src/index.html

```diff
@@ -7,7 +7,61 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Stars</h1>
+
+    <div class="container">
+
+      <div class="stars stars--0">
```

Comment:
```suggestion
    <div class="container">
      <div class="stars stars--0">
```
Don't leave empty lines between parent and child elements